src/redux-state/thunks/island.ts

```diff
@@ -95,7 +96,10 @@ export const initSeasonInfoData = createDappAsyncThunk(
 
 export const fetchPopulation = createDappAsyncThunk(
   "island/fetchPopulation",
-  async (_, { getState, dispatch, extra: { transactionService } }) => {
+  async (
+    isInitFetch: boolean,
```

Comment:
this is not my fav solution 😅 will think about suggesting something else later, maybe checking if realm displayed population in redux is already set would be better than using this argument?